internal/learn/the-community/qanda.md

```diff
@@ -0,0 +1,98 @@
+
+The IOTA technology with the digital marketplaces enables a free, secure and trustless linking of unknown business partners. The technology offers a very cost effective or even free possibility to link the chain of events of an order in a single system at the correct time. If one were to outsource individual events, e.g. external payment without IOTA, it would be an additional effort to enter this individual event back into the event chain of the marketplace at the correct time. This would cause additional costs compared to the pure IOTA solution and the IOTA solution would therefore have competitive advantages.
+
+**Cross-country and cross-company exchange medium**
+If you want to map cross-country and cross-company processes, trade or logistics between machines these transactions can take place without much delay and without having to agree on a trading currency etc. The only requirement is that machines have direct access to a stock exchange either directly or via a financial service provider to make conversions. This could be, for example, the house bank of a company, which keeps the Fiat accounts anyway. If the bank charges a small margin for this autonomous conversion job this is absolutely legitimate. A machine could autonomously submit a conversion order to the bank at any time and then have its IOTA tokens available to send them across the national border. In this case, the higher volatility is completely neglected and a machine would not have to keep tokens in its wallet permanently.
+
+> IOTA is an open, free of charge, not centralized and ultra-fast clearing system!
+
+**BTC**
+For years, the BTC experts have been saying that you can pay for everything with BTC. Among other things, this would bring all the disadvantages of BTC into decentralized trading such as incalculable transaction costs, long confirmation times, inaccurate timestamps and high energy costs. The IOTA token, which is firmly integrated into the protocol, was created to eliminate all these shortcomings and to enable real micro-transactions between machines. In addition, a BTC timestamp with an accuracy of 2 hours is completely unsuitable for the machine economy where transactions may occur every second. For example, the payment at an e-charging station could be done via IOTA Streams. For this purpose, a private IOTA Streams channel is opened between the vehicle and the e-charging point to establish a data stream. Payment is made per data point in micro transactions and payment stops as soon as no more power is received. This does not work with an external coin like BTC.
```

Comment:
```suggestion
BTC experts have been saying that you can pay for everything with BTC for years. Among other things, this would bring all the disadvantages of BTC into decentralized trading, such as incalculable transaction costs, long confirmation times, inaccurate timestamps, and high energy costs. The IOTA token, firmly integrated into the protocol, was created to eliminate all these shortcomings and enable real micro-transactions between machines. In addition, a BTC timestamp with an accuracy of 2 hours is completely unsuitable for the machine economy, where transactions may occur every second. For example, the payment at an e-charging station could be made via IOTA Streams. For this purpose, a private IOTA Streams channel is opened between the vehicle and the e-charging point to establish a data stream. Payment is made per data point in micro-transactions, and payment stops as soon as no more power is received. This does not work with an external coin like BTC.
```